assembly/pom.xml

```diff
@@ -624,7 +624,24 @@
                                                 </source>
                                             </sources>
                                         </mapping>
+                                        <mapping>
+                                            <directory>/var/lib/splicemachine</directory>
+                                            <sources>
+                                                <source>
+                                                    <location>common/src/main/resources/bin/sqlshell.sh</location>
+                                                </source>
+                                            </sources>
+                                            <filemode>
+                                                755
+                                            </filemode>
+                                            <directoryIncluded>
+                                                false
+                                            </directoryIncluded>
+                                        </mapping>
                                     </mappings>
+                                    <postinstallScriptlet>
+                                      <script>sed -i 's/##SPLICELIBDIR##/\/var\/lib\/splicemachine/g' /var/lib/splicemachine/sqlshell.sh</script>
```

Comment:
one technique to make this sed easier to read, is use a different statment separating character, instead of / which then has to be escaped, use e.g. @. 

's@##SPLICELIBDIR##@/var/lib/splicemachine@g' /var/lib/splicemachine/sqlshell.sh